templates/soap/ConsumeSOAPExample/ConsumeSOAPExample.cs

```diff
@@ -5,32 +5,49 @@
 using ConsumeSOAPExample.Structures;
 
 namespace ConsumeSOAPExample{
-    /* In this case, the OSInterface attribute is used to provide information about the Calculator interface to the ODC, 
-        which will use this information to create an integration layer with a SOAP web service. */
+    /// <summary>
+    /// The ICalculator interface defines the methods (exposed as server actions)
+    /// for basic mathematical operations. These server actions are essentially
+    /// SOAP-based web service operations exposed in a .NET context.
+    /// </summary>
     [OSInterface(Description = "Consume a Free SOAP web service in OutSystems Developer Cloud (ODC).", IconResourceName = "ConsumeSOAPExample.Resources.SOAP.jpg", Name = "SOAP")]
     public interface ICalculator
     {
-        // This method is exposed as a server action to your ODC apps and libraries.
+        /// <summary>
+        /// Takes two numbers as input parameters and returns their sum. This is done by
+        /// sending a SOAP request to the web service, which then processes the addition 
+        /// and returns the result in the SOAP response.
+        /// This method is exposed as a server action to your ODC apps and libraries.
+        /// </summary>
         [OSAction(Description = "The sum method takes two numbers as input parameters and returns their sum.", IconResourceName = "ConsumeSOAPExample.Resources.SOAP.jpg", ReturnName = "sum")]
         int Sum(
             [OSParameter(Description = "Number A & B as an integer")] Structures.Numbers numbers
         );
-        // This method is exposed as a server action to your ODC apps and libraries.
+        /// <summary>
+        /// Takes two numbers as input parameters and calculates the difference between 
```

Comment:
Suggested change: and returns the difference between

Reply:
Agred, changed.